olm_deploy/operatorregistry/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM quay.io/operator-framework/upstream-registry-builder AS registry-builder
 FROM registry.ci.openshift.org/ocp/4.7:base
 
 WORKDIR /
-COPY manifests/ /manifests
+COPY bundle/manifests/ /manifests
```

Comment:
@periklis same change should go here too https://github.com/openshift/elasticsearch-operator/blob/master/Dockerfile#L19
The tests are failing because of no change in above file.